deps.edn

```diff
@@ -230,7 +230,7 @@
     djblue/portal                {:mvn/version "0.51.0"}                    ; ui for inspecting values
     hashp/hashp                  {:mvn/version "0.2.2"}                     ; debugging/spying utility
     io.github.camsaul/humane-are {:mvn/version "1.0.2"}
-    io.github.metabase/hawk      {:sha "46835a0b808e86894f598472351c08d81aeb1910"}
+    io.github.metabase/hawk      {:local/root "../hawk"}
```

Comment:
Waiting on https://github.com/metabase/hawk/pull/19 to be approved, then I will update this.

Reply:
✅ on that pr.